### PR TITLE
Refactor code - Retry session timer

### PR DESCRIPTION
**Pull request by alber2510**

### CODE REVIEW

File: src/rtcModule/webrtc.cpp [hunk 1]
```diff
@@ -823,13 +817,8 @@ void Call::msgCallTerminate(RtMessage& packet)
     {
         EndpointId endpointId(packet.userid, packet.clientid);
         time_t sessionRetryTime = 0;
-        auto itSessionRetyTime = mSessRetriesTime.find(endpointId);
-        if (itSessionRetyTime != mSessRetriesTime.end())
-        {
-            sessionRetryTime = itSessionRetyTime->second;
-        }
-
-        if (sessionRetryTime && (time(NULL) - sessionRetryTime <= RtcModule::kSessSetupTimeout))
```
Comment (sergiohs84) [inline src/rtcModule/webrtc.cpp]: Does it mean we don't need the timeout for session-setup any longer?

File: src/rtcModule/webrtc.cpp [hunk 2]
```diff
@@ -734,18 +734,12 @@ void Call::handleMessage(RtMessage& packet)
             TermCode reason = static_cast<TermCode>(data.read<uint8_t>(8));
             chatd::EndpointId endPointId(packet.clientid, packet.userid);
 
-            auto itRetriesTime = mSessRetriesTime.find(endPointId);
-            if (!Session::isTermRetriable(reason) && itRetriesTime != mSessRetriesTime.end())
+            auto itRetries = mSessRetries.find(endPointId);
+            if (!Session::isTermRetriable(reason) && itRetries != mSessRetries.end())
```
Comment (sergiohs84) [inline src/rtcModule/webrtc.cpp]: Shouldn't we code it like below for homogeneity?
``` if (!Session::isTermRetriable(reason)
{
    cancelSessionRetryTimer(packet.userid, packet.clientid);
}```

File: src/rtcModule/webrtc.cpp (hunk 1 above)
Comment (alber2510) [reply src/rtcModule/webrtc.cpp]: The timeout is controlled by mSessRetries (std::map<chatd::EndpointId, megaHandle>). If pair (EndpointId, timer handle) exists, there is a session retry in course. When timeout expires, the handle is removed from map

File: src/rtcModule/webrtc.cpp (hunk 2 above)
Comment (alber2510) [reply src/rtcModule/webrtc.cpp]: I've thought but _destroyIfNoSessionsOrRetries(reason)_  has to execute if both conditions are valid, _!Session::isTermRetriable(reason)_ and _itRetries != mSessRetries.end()_
Then I thought that it is more simple doing without using _cancelSessionRetryTimer_
If you considerate opportune I can change it

**Pull request merged**